email/1.3/modules/ROOT/edits/email-studio-configuration.adoc

```diff
@@ -0,0 +1,147 @@
+= Use Studio to Configure Email Connector 1.3 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:email-documentation.adoc[Email Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Email Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `email` in the search field.
+. Click *Email Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Email Connector:
+
+* *On New Email - IMAP* +
+Initiates a flow by retrieving all the emails from an IMAP mailbox folder, watermark can be enabled for polled items
+* *On New Email - POP3* +
+Initiates a flow by retrieving all the emails from an POP3 mailbox folder
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
```

Comment:
We have to add periods because of the one description that includes a complete sentence, per style ref.
```suggestion
Initiates a flow each time it receives a request on the configured host and port.
```